.travis.yml

```diff
@@ -196,8 +196,8 @@ before_script:
   - conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
 
 build_cyclonedds: &build_cyclonedds
-  - eval "export CYCLONEDDS_BRANCH=master"
```

Comment:
Should be set back to master or he origin repo once the branch is merged there.

Reply:
Yep.